File: snippets/fsharp/System/String/op_Inequality/inequalityop.fs
module inequalityop.fs
//<snippet1>
// Example for the String Inequality operator.
printfn "This example of the String Inequality operator\ngenerates the following output.\n"

let compareAndDisplay comparand =
    let lower = "abcd"
    printfn $"\"%s{lower}\" <> \"%s{comparand}\" ?  {lower <> comparand}"

compareAndDisplay "ijkl"
compareAndDisplay "ABCD"
compareAndDisplay "abcd"

(*
This example of the String Inequality operator
generates the following output.

"abcd" <> "ijkl" ?  True
"abcd" <> "ABCD" ?  True
"abcd" <> "abcd" ?  False
*)
//</snippet1>

File: snippets/csharp/System/String/op_Inequality/inequalityop.cs
//<snippet1>
// Example for the String Inequality operator.
using System;

class InequalityOp
{
    public static void Main()
    {
        Console.WriteLine(
            "This example of the String Inequality operator\n" +
            "generates the following output.\n");

        CompareAndDisplay("ijkl");
        CompareAndDisplay("ABCD");
        CompareAndDisplay("abcd");
    }

    static void CompareAndDisplay(string Comparand)
    {
        string Lower = "abcd";

        Console.WriteLine($"\"{Lower}\" != \"{Comparand}\" ?  {Lower != Comparand}");
    }
}

/*
This example of the String Inequality operator
generates the following output.

"abcd" != "ijkl" ?  True
"abcd" != "ABCD" ?  True
"abcd" != "abcd" ?  False
*/
//</snippet1>
